policy/pa.go

```diff
@@ -161,6 +161,10 @@ const (
 	maxDNSIdentifierLength = 230
 )
 
+// The values of maxDNSIdentifierLength, maxLabelLength and maxLabels are hard coded
```

Comment:
I think this comment should be placed before the const definitions instead of after them, probably on line 147 before the `const (`.

Reply:
Moved